--- Mail-ClamAV-0.29/Makefile.PL	2009-04-29 15:06:39.000000000 -0400
+++ Mail-ClamAV-0.29/Makefile.PL2	2018-10-22 09:16:37.809713900 -0400
@@ -18,7 +18,7 @@
     return $out;
 }
 
-my $minclversion = "0.95.1";  # minimum clamav lib version
+my $minclversion = "0.100.1";  # minimum clamav lib version
 my $DEFAULT_LIBS = q(-lz -lbz2 -lgmp -lpthread);
 my $DEFAULT_INC = '-I/usr/include';
 my $AUTHOR = 'David P.C. Wollmann <converter42@gmail.com>';
